test/EvaluationTests/EvaluationTests.cs

```diff
@@ -0,0 +1,212 @@
+            if (string.IsNullOrEmpty(connectionString))
+            {
+                throw new InvalidOperationException($"Missing connection string {connectionStringName}");
+            }
+
+            connectionStringBuilder.ConnectionString = connectionString;
+
+            var deployment = connectionStringBuilder.TryGetValue("Deployment", out var deploymentValue) ? (string)deploymentValue : throw new InvalidOperationException($"Connection string {connectionStringName} is missing 'Deployment'");
+            var endpoint = connectionStringBuilder.TryGetValue("Endpoint", out var endpointValue) ? (string)endpointValue : throw new InvalidOperationException($"Connection string {connectionStringName} is missing 'Endpoint'");
+            //var key = connectionStringBuilder.TryGetValue("Key", out var keyValue) ? (string)keyValue : throw new InvalidOperationException($"Connection string {connectionStringName} is missing 'Key'");
+
+            return new AzureOpenAIClient(new Uri(endpoint), new DefaultAzureCredential()).AsChatClient(deployment);
+        }
+
+        static ReportingConfiguration GetReportingConfiguration()
+        {
+            // Setup and configure the evaluators you would like to utilize for each AI chat
+            IEvaluator rtcEvaluator =
+                new RelevanceTruthAndCompletenessEvaluator(
+                    new RelevanceTruthAndCompletenessEvaluator.Options(includeReasoning: true));
+            IEvaluator coherenceEvaluator = new CoherenceEvaluator();
+            IEvaluator fluencyEvaluator = new FluencyEvaluator();
+            IEvaluator groundednessEvaluator = new GroundednessEvaluator();
+            IEvaluator answerScoringEvaluator = new AnswerScoringEvaluator();
+
+            var endpoint = new Uri(AzureOpenAIEndpoint);
+            var azureClient = new AzureOpenAIClient(endpoint, new DefaultAzureCredential());
+
+            IChatClient chatClient = azureClient.AsChatClient(AzureOpenAIDeploymentName);
+            Tokenizer tokenizer = TiktokenTokenizer.CreateForModel(AzureOpenAIModelName);
+
+            var chatConfig = new ChatConfiguration(chatClient, tokenizer.ToTokenCounter(6000));
+
+            return DiskBasedReportingConfiguration.Create(
+                    storageRootPath: StorageRootPath,
+                    chatConfiguration: chatConfig,
+                    evaluators: [
+                        rtcEvaluator,
+                        coherenceEvaluator,
+                        fluencyEvaluator,
+                        groundednessEvaluator,
+                        answerScoringEvaluator],
+                    executionName: ExecutionName);
+        }
+
+        const string StorageRootPath = @"C:\src\eShopCache";
+        const string AzureOpenAIEndpoint = "https://aievaluation-openai.openai.azure.com/";
+        const string AzureOpenAIModelName = "gpt-4o";
+        const string AzureOpenAIDeploymentName = "gpt-4o";
```

Comment:
It may be better to fetch these consts from environment variables. At least I think we should avoid hardcoding the endpoint url for the internal subscription...

Reply:
Also wondering if the evaluations should also use Ollama by default similar to the main sample?

Reply:
> It may be better to fetch these consts from environment variables. At least I think we should avoid hardcoding the endpoint url for the internal subscription...

Good catch, I intended to change that earlier. I've updated the PR to read these settings from `appsettings.json` and put placeholders in the file.

> Also wondering if the evaluations should also use Ollama by default similar to the main sample?

This could be interesting but given that I have no way to run Ollama for the main sample right now, this will have to come later.